pandas/tests/indexing/test_iloc.py

```diff
@@ -1363,6 +1382,25 @@ def test_frame_iloc_setitem_callable(self):
         exp.iloc[[1, 3], [0]] = [-5, -5]
         tm.assert_frame_equal(res, exp)
 
+    def test_frame_iloc_setitem_callable_multicolumn_to_datetime(
```

Comment:
I don't think this test is needed? It looks similar to the other test.

Reply:
I have used apply to convert to datetime. Yeah it's kind of look simillare. Removed